### ISSUE TITLE
lost alignment between columns

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#78]
Hi,

I've found a little problem with top aligned columns in a frame. Please, test attached example. You will see that itemized environments are not aligned when the frame has `[t]` option and they look good with the centered frame default option.

Do you know what's wrong?

Thank you,

Ignasi

````
\documentclass{beamer}
\begin{document}
\begin{frame}[t]{Test Columns}
\begin{columns}
\column{.5\linewidth}
First Column
\begin{itemize}
\item First Item First Item First Item First Item
\item Second Item Second Item Second Item Second Item
\end{itemize}
\column{.5\linewidth}
Second Column
\begin{itemize}
\item First Item First Item First Item First Item
\item Second Item Second Item Second Item Second Item
\end{itemize}
\end{columns}
\end{frame}
\begin{frame}{Test Columns}
\begin{columns}
\column{.5\linewidth}
First Column
\begin{itemize}
\item First Item First Item First Item First Item
\item Second Item Second Item Second Item Second Item
\end{itemize}
\column{.5\linewidth}
Second Column
\begin{itemize}
\item First Item First Item First Item First Item
\item Second Item Second Item Second Item Second Item
\end{itemize}
\end{columns}
\end{frame}
\end{document}
````

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/78
- Originally reported by: Ignasi  
- Originally created at: 2010-09-21

**Comment by josephwright** [comment]
_From Vedran Miletić on 2010-10-26_

I know how to fix it, and it's quite easy, but I'm not sure if it will break something else. If you need the fix badly, replace lines 256--260 in `beamerbaseframe.sty` which read:
````
  \def\beamer@initfirstlineunskip{%
    \def\beamer@firstlineitemizeunskip{%
      \vskip-\partopsep\vskip-\topsep\vskip-\parskip%
      \global\let\beamer@firstlineitemizeunskip=\relax}%
    \everypar{\global\let\beamer@firstlineitemizeunskip=\relax}}
````
with
````
  \def\beamer@initfirstlineunskip{}%
````
This will fix it.

**Comment by josephwright** [comment]
_From svenkoehler on 2013-10-01_

Why is this still on hold? I just hit this bug and an itemize in columns on a top-align frame really looks very bad!

A workaround that can be copy/pasted into the document itself is the following:
````
\makeatletter
%%% taken from beamerbaseframe.sty
\define@key{beamerframe}{t}[true]{% top
  \beamer@frametopskip=.2cm plus .5\paperheight\relax%
  \beamer@framebottomskip=0pt plus 1fill\relax%
  \beamer@frametopskipautobreak=\beamer@frametopskip\relax%
  \beamer@framebottomskipautobreak=\beamer@framebottomskip\relax%
  \def\beamer@initfirstlineunskip{}%
}
\makeatother
````
which was suggested here: http://tex.stackexchange.com/questions/16447/beamer-top-aligning-columns-within-a-top-aligned-frame

**Comment by josephwright** [comment]
_From Joseph Wright on 2013-10-01_

I'll take a look at this

**Comment by josephwright** [comment]
Fixed by 5658a3ee52b77f7f1e87668630bc096115030c92
